canonical-web-and-design/vanilla-framework
# WIP: Updates to application layout panels responsiveness

## Done

- make sure aside panel gets unpinned on small screens
- make sure aside panel doesn't exceed small screen width
- provide 3 width options for panels (is-narrow, is-medium, is-wide)

Fixes #3283

## QA

- Run `./run` or [demo](https://vanilla-framework-3382.demos.haus/docs/examples/layouts/application)
- Check application layout panels example https://vanilla-framework-3382.demos.haus/docs/examples/layouts/application
  - make sure aside panel works well responsively (unpins on small screens, is never wider than the screen)
  - check different built-in widths of the panel
